src/engine/world/actor/walking-queue.ts

```diff
@@ -144,6 +151,12 @@ export class WalkingQueue {
 
         const walkPosition = this.queue.shift();
 
+        if (!walkPosition) {
+            // (Jameskmonger) not sure if this is an issue or not..
+            logger.warn('No walk position found');
```

Comment:
This should be an unreachable branch since if the queue length is 0, we a returning early